### ISSUE TITLE
Release 0.1.3 missing properties

### ISSUE THREAD

**Issue by grbd** [FortAwesome/vue-fontawesome#161]
Hi,
I've been having some issues when moving from 0.1.2 to 0.1.3

```
ERROR in D:\SourceCode\GitRepos\GBD.WebTemplates.AspNetCore\src\AspNetCore.Vue.Foundation1\ClientApp\common\svgicon.ts
./ClientApp/common/svgicon.ts
[tsl] ERROR in D:\SourceCode\GitRepos\GBD.WebTemplates.AspNetCore\src\AspNetCore.Vue.Foundation1\ClientApp\common\svgicon.ts(13,30)
      TS2345: Argument of type 'FunctionalComponentOptions<Record<string, any>, PropsDefinition<Record<string, any>>>' is not assignable to parameter of type 'ComponentOptions<Vue, DefaultData<Vue>, DefaultMethods<Vue>, DefaultComputed, PropsDefinition<Record<string, any>>, Record<string, any>>'.
  Types of property 'render' are incompatible.
    Type '((this: undefined, createElement: CreateElement, context: RenderContext<Record<string, any>>) => VNode | VNode[]) | undefined' is not assignable to type '((createElement: CreateElement, hack: RenderContext<Record<string, any>>) => VNode) | undefined'.
      Type '(this: undefined, createElement: CreateElement, context: RenderContext<Record<string, any>>) => VNode | VNode[]' is not assignable to type '(createElement: CreateElement, hack: RenderContext<Record<string, any>>) => VNode'.
        Type 'VNode | VNode[]' is not assignable to type 'VNode'.
          Type 'VNode[]' is missing the following properties from type 'VNode': isRootInsert, isComment
```

Typically the way I'm using it is
```
import Vue from "vue";
import { faCog, faFile, faFolder, faFolderOpen, faSpinner } from "@fortawesome/free-solid-svg-icons";
import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";

library.add(faSpinner, faCog, faFolder, faFolderOpen, faFile);
Vue.component("fa-icon", FontAwesomeIcon);
```

This works fine under 0.1.2 but breaks under 0.1.3 <br>
As an example code base

  * https://github.com/grbd/GBD.WebTemplates.AspNetCore/tree/master/src/AspNetCore.Vue.Foundation1

I'm guessing it has something to do with the addition of the types for typescript

**Comment by sps-smith** [comment]
I am experiencing the same issue. I am using Vuejs with a SharePoint framework solution for office 365. Everything worked fine in a previous solution using 0.1.2 but failed with 0.1.3


```
import { library } from '@fortawesome/fontawesome-svg-core';
import { faArrowCircleUp } from '@fortawesome/free-solid-svg-icons';
import { faArrowCircleDown } from '@fortawesome/free-solid-svg-icons';
import {FontAwesomeIcon } from '@fortawesome/vue-fontawesome';


library.add(faArrowCircleUp,faArrowCircleDown);

Vue.component('font-awesome-icon', FontAwesomeIcon);`
```
Package.json file
```
  "dependencies": {
    "@fortawesome/fontawesome-svg-core": "^1.2.10",
    "@fortawesome/free-solid-svg-icons": "^5.6.1",
    "@fortawesome/vue-fontawesome": "^0.1.3",
    "@microsoft/decorators": "1.7.0",
    "@microsoft/sp-application-base": "1.7.0",
    "@microsoft/sp-core-library": "1.7.0",
    "@microsoft/sp-dialog": "1.7.0",
    "@pnp/pnpjs": "^1.2.5",
    "@pnp/spfx-property-controls": "1.12.0",
    "@types/es6-promise": "0.0.33",
    "@types/jquery": "^3.3.22",
    "@types/webpack-env": "1.13.1",
    "bootstrap-vue": "^2.0.0-rc.11",
    "jquery": "^3.3.1",
    "vue": "^2.5.17",
    "vue-class-component": "^6.3.2",
    "vue-property-decorator": "^7.2.0"
  },
```

**Comment by Alexander-Karsten** [comment]
Also experiencing the same issue here

**Comment by eakarpov** [comment]
Me too.

**Comment by robmadole** [comment]
@schulz3000 submitted a PR that hopefully will fix the issue. I've just released 0.1.4 to try and address this.

**Comment by meteorlxy** [comment]
I added types declaration files in 0.1.3.

That wroked fined with Vue 2.5.17, but failed with Vue 2.5.18+ due to https://github.com/vuejs/vue/compare/v2.5.17...v2.5.18#diff-23d7799dcc9e9be419d28a15348b0d99L116

I tested on Vue 2.5.17 and did not meet this error. 😅 Sorry for introduced this issue.
